pkg/config/stepmeta.go

```diff
@@ -47,6 +47,13 @@ type StepParameters struct {
 	Type            string      `json:"type"`
 	Mandatory       bool        `json:"mandatory,omitempty"`
 	Default         interface{} `json:"default,omitempty"`
+	Aliases         []Alias     `json:"aliases,omitempty"`
+}
+
+// Alias defines a step input parameter alias
+type Alias struct {
+	Name       string `json:"name,omitempty"`
+	Deprecated bool   `json:"deprecated,omitempty"`
```

Comment:
The `Deprecated´ flag seems not to be used (neither in the productive code nor are there any tests for it). Is that a left-over?

Reply:
Currently it purely targets generation of documentation. We might enforce additional things as soon as we having logging in place.